assets/methods_description_template.yml

```diff
@@ -0,0 +1,25 @@
+id: "nf-core-nascent-methods-description"
+description: "Suggested text and references to use when describing pipeline usage within the methods section of a publication."
+section_name: "nf-core/nascent Methods Description"
+section_href: "https://github.com/nf-core/nascent"
+plot_type: "html"
+## TODO nf-core: Update the HTML below to your prefered methods description, e.g. add publication citation for this pipeline
```

Comment:
```suggestion

```

Reply:
Leaving this as a reminder to add later, unless you think I should remove it.